hibernate-core/src/main/java/org/hibernate/boot/registry/selector/internal/StrategySelectorBuilder.java

```diff
@@ -166,7 +162,7 @@ public StrategySelector buildSelector(ClassLoaderService classLoaderService) {
 		// build the baseline...
 		addDialects( strategySelector );
 		addJtaPlatforms( strategySelector );
-		addTransactionFactories( strategySelector );
+//		addTransactionFactories( strategySelector );
```

Comment:
I think we still want some short naming here for TransactionCoordinator impls...